code/modules/mob/living/simple_animal/borer/egg.dm

```diff
@@ -98,6 +101,13 @@
 	else
 		..()
 
+/obj/item/weapon/reagent_containers/food/snacks/borer_egg/attack_ghost(var/mob/dead/observer/O)
+	if(last_ping_time + ping_cooldown <= world.time)
+		visible_message(message = "<span class='notice'>\The [src] wriggles vigorously.</span>", blind_message = "<span class='danger'>You hear what you think is someone jiggling a jelly.</span>")
+		last_ping_time = world.time
+	else
```

Comment:
Doesn't really need an else statement calling people cunts, if two people try clicking it after one another it's gonna trigger. Just have it return.

Also, please add a cooldown to posibrains https://github.com/d3athrow/vgstation13/blob/a559bcf4e9f3004ad5969e1344680db1f37575a2/code/modules/mob/living/carbon/brain/posibrain.dm#L170 since you're doing it here.

And 15 seconds is still insanely large, do something like 5.